llvm/test/Instrumentation/InstrProfiling/section-code-model-large.ll

```diff
@@ -0,0 +1,35 @@
+;; Check that certain globals are in large sections under x86-64 large code model (but not in other arches).
+; RUN: opt %s -mtriple=x86_64-unknown-linux -passes=instrprof -S | FileCheck %s
```

Comment:
The comment makes it sound like this should have a RUN line for other arches. Was that intentional or copy paste from the other tests?